File: LegoSCart/asplego/Models/Payments.cs
using System;
namespace Asplego.Models
{
    public class Payment
    {
        public int Cardnumber;
        public string Bankname;
        public string Holdname;
        public string Expiredate;
        public int CVS;

        public Payment(int cardnumber, string bankname, string holdname, string exp, int cvs)
        {

            Cardnumber = cardnumber;
            Bankname = bankname;
            Holdname = holdname;
            Expiredate = exp;
            CVS = cvs;


        }

    }
}

File: LegoSCart/LegoShop/src/app/shared/models/Payments.ts
export class Payment{
      
    Cardnumber: number;
    Bankname: string;
    Holdname: string;
    Expiredate: string;
    CVS: number;
   
    constructor(cardnumber:number,bankname:string,holdname:string,expiredate:string,cvs:number){
        this.Cardnumber = cardnumber;
        this.Bankname = bankname;
        this.Holdname = holdname;
        this.Expiredate = expiredate;
        this.CVS=cvs;

  }
    
}
